src/h3lib/lib/h3Index.c

```diff
@@ -355,7 +355,7 @@ int H3_EXPORT(compact)(const H3Index* h3Set, H3Index* compactedSet,
         memset(hashSetArray, 0, numHexes * sizeof(H3Index));
         compactedSetOffset += uncompactableCount;
         memcpy(remainingHexes, compactableHexes,
-               maxCompactableCount * sizeof(H3Index));
+               compactableCount * sizeof(H3Index));
```

Comment:
This was causing a clang-tidy warning. It seems `maxCompactableCount` is less precise than `compactableCount` for specifying the size to copy. Relates to accessing array out of bounds.